### ISSUE TITLE
250 Listings for Craigslist, Jayanta Data Entry Project. To Be Completed no later than 3/17

### ISSUE THREAD

**Issue by urentmellc** [pixelphpdev/urentme#20]
Jayanta,
 
Please see below criteria for doing the 250 listings. Id like them done by next weeks end… You have the video and stated you have understood all to me. Please do 20 tomorrow your Saturday and I will check them and advise. You will need to copy the description, do not copy the for sale info as its not for sale. Be sure to note this or it will require full revision. Please advise if needed. Roughly 12 ish listings per category per city. Do not choose items older that 2008. Thank you.
 
Cities
Las Vegas-50 listings
San Diego-50 “
Mohave County-50 “
Phoenix-50 “
SF Bay Area-50 “
 
Catagories on CL
ATV/UTV
RV’s Camp
Boats
Motorcycles

**Comment by urentmellc** [comment]
this is on hold until site bugs are resolved...

**Comment by Jayantapixel** [comment]
Please re-confirm here so we can do at least 20 items tomorrow and show you. If all ok, then will do rest from Monday and complete quickly.

**Comment by Jayantapixel** [comment]
Just added one item. Primary details as attached.
I have a query:
Do I need to add/create 250 users to enter 250 items?  OR, we can put 5-6 items per user also. 
Let me know.
[URentMe Data Entry.xlsx](https://github.com/pixelphpdev/urentme/files/1819305/URentMe.Data.Entry.xlsx)

**Comment by urentmellc** [comment]
I did however still finding bugs in the registration and listing concerns… will update in github… 

 

Anthony HorovitzFounder, URentMe.Com855-400-0082 | 702-219-0186 | anthony@urentme.com | www.urentme.com | Skype: anthony.horovitz | 871 Coronado Center Drive, Suite 200, Henderson, NV 89052

**Comment by urentmellc** [comment]
All bugs in system must be addressed before we can continue with this... new issues have been found.
